### PR TITLE
src: refactor mod.ts

### PR DESCRIPTION
**Pull request by vierofernando**
changed how the client validates IDs, and improved switch statement.

### CODE REVIEW

File: src/mod.ts [hunk 1]
```diff
@@ -158,10 +158,7 @@ export class Webhook extends EventEmitter<Events> {
 							const data: WebhookVote = JSON.parse(new TextDecoder().decode(buffer));
 							this.emit('vote', data);
 						});
-
-						req.respond({ status: 200, body: '' });
```
Comment (VoltrexMaster) [inline src/mod.ts:162]: Was there a reason this was removed?

Comment (vierofernando) [reply src/mod.ts:162]: i made it so the code will also run the `default:` keyword (since there are no `break` statements in the `case` statement)

Comment (VoltrexMaster) [reply src/mod.ts:162]: Oh alright.

**Review by VoltrexMaster — APPROVED** [review]
❤️

**Comment by vierofernando** [comment]
❤️

**Pull request merged**